src/pytest_cov/plugin.py

```diff
@@ -73,7 +73,7 @@ def pytest_addoption(parser):
     group.addoption('--no-cov', action='store_true', default=False,
                     help='Disable coverage report completely (useful for debuggers). '
                          'Default: False')
-    group.addoption('--cov-fail-under', action='store', metavar='MIN', type=int,
+    group.addoption('--cov-fail-under', action='store', metavar='MIN', type=float,
```

Comment:
```python
import ast
import numbers

def parse_fail_under(num_str):
    try:
        num = ast.literal_eval(num_str)
    except SyntaxError:
        raise ValueError
    if isinstance(num, numbers.Real):
        return num
    raise ValueError
```

how about this. So that 1.0 and 1 do not behave the same

Reply:
that looks simple and elegant. +1.